packages/@webex/plugin-meetings/test/unit/spec/meeting/brbState.ts

```diff
@@ -104,12 +104,19 @@ describe('plugin-meetings', () => {
       assert.isTrue(meeting.meetingRequest.setBrb.calledOnce);
     });
 
-    it('sets source state override when client state does not match server state', async () => {
+    it('update source state override when client state does not match server state', async () => {
```

Comment:
Nit: change "update" to "updates", so this reads "it updates source state override".